source/languages/en/riakcs/cookbooks/Riak-CS-Release-Notes.md

```diff
@@ -34,190 +74,159 @@ Notes][riak_cs_2.0.0_release_notes]**
+For the Stanchion `app.config`:
 
-Instructions follow:
+ - `stanchion_ip` and `stanchion_port` have been combined into `listener`.
+ - `riak_ip` and `riak_port` have been combined into `riak_host`.
 
-1. Stop the Riak CS process
-2. Backup configuration files
-3. Uninstall the current Riak CS package
-4. Install the Riak CS 2.0.0 package
-5. Update configuration
-6. Start Riak CS
+Each of the above pairs follows a similar form. For example, if your legacy
+`app.config` configuration was previously:
 
-#### Configuration upgrade
+```
+{riak_cs, [
+    {cs_ip, "127.0.0.1"},
+    {cs_port, 8080 },
+    . . .
+]},
+```
 
-Riak 2.0.0 introduced a new configuration system (`riak.conf`). It still
-supports the older style configurations through `app.config` and `vm.args`. Riak
-CS 2.0.0 now follows suit and supports the new configuration style.
+It should now read:
 
-Basho recommends moving to the new unified configuration system and use the
-files `riak.conf`, `riak-cs.conf` and `stanchion.conf`.
+```
+{riak_cs, [
+    {listener, {"127.0.0.1", 8080}},
+    . . .
+]},
+```
 
-**For a complete set of changed configuraion settings please see the [Full Riak
-CS 2.0.0 Release Notes][riak_cs_2.0.0_release_notes]**
+and so on.
 
-##### Riak 1.4.x to Riak 2.0.5
+#### Note: Upgrading from Riak CS 1.5.3 or Older
 
-As Riak CS 2.0.0 only works on top of Riak 2.0.5 -- and does *not* work on top
-of the Riak 1.x.x series -- the underlying Riak installation *must* be upgraded
-to Riak 2.0.5. General guides for upgrading Riak to 2.0.5 are in the
-[Upgrading to 2.0 guide][upgrading_to_2.0].
+[Some key objects changed names][riak_cs_1.5_release_notes_upgrading] after the
+upgrade. Applications may need to change their behaviour due to this bugfix.
 
-Below are configuration changes for a Riak cluster supporting Riak CS.
+#### Note: Upgrading from Riak CS 1.5.0 or Older
 
-##### Review memory size
+[Bucket number limitation per user][riak_cs_1.5_release_notes_upgrading_1] have
+been introduced in 1.5.1. Users who have more than 100 buckets cannot create any
+bucket after the upgrade unless the limit is extended in the system
+configuration.
 
-Since the default configuration of the LevelDB memory size has changed, review
-your memory size settings. The memory use of Riak CS is dominated by the Bitcask
-keydir and LevelDB block cache. Additionally, to improve IO performance, some
-extra memory for the kernel disc cache should be configured. The equations below
-might help when specifying the memory size:
+#### Note: Upgrading From Riak CS 1.4.x
 
-```
-Memory for storage = (Memory for backends) + (Memory for kernel cache)
-Memory for backends = (Memory for Bitcask) + (Memory for LevelDB)
-```
+An operational procedure [to clean up incomplete multipart under deleted
+buckets][riak_cs_1.5_release_notes_incomplete_mutipart] is needed. Otherwise new
+buckets with names that used to exist in the past can't be created. The
```

Comment:
"used to exist" and "in the past" are redundant. Suggest dropping "in the past".